### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Static Content: No label name is provided for a control(Text Field)

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#157]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
**WCAG ID Violated:** WCAG3.3.2

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #C+E-H; #.NETWebsites; #Chrome; #Edge; WCAG3.3.2

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Static Content and press enter
6.	Validate whether all the controls have labels or not

**Actual Result:**
A text field doesn’t have a label name

**Expected Result:**
All controls should have a label name

**User Impact:**
Users with disabilities may not understand which data to fill in the text field if label name is not provided

**Attaching screenshot for your reference:**
![bug 157_label name](https://user-images.githubusercontent.com/42638744/44772582-e4122b80-ab8b-11e8-9ce3-3a6ca33a83cd.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
